csharp/test/Apache.Arrow.Tests/TestData.cs

```diff
@@ -176,6 +179,21 @@ public void Visit(StringType type)
                 Array = builder.Build();
             }
 
+            public void Visit(ListType type)
+            {
+                //Todo : Use ListArray.Builder
```

Comment:
I'll fix this at [ARROW-7290: [C#] Implement ListArray Builder](https://issues.apache.org/jira/browse/ARROW-7290)